lib/bots/versionbot.ts

```diff
@@ -769,9 +769,9 @@ export class VersionBot extends ProcBot {
         };
 
         return Promise.mapSeries([
-            `git clone https://${versionData.authToken}:${versionData.authToken}@github.com/` +
-                `${versionData.repoFullName} ${versionData.fullPath}`,
-            `git checkout ${versionData.branchName}`
+            `git clone "https://${versionData.authToken}:${versionData.authToken}@github.com/` +
+                `${versionData.repoFullName}" ${versionData.fullPath}`,
+            `git checkout "${versionData.branchName}"`
```

Comment:
Actually the right way to do this is to ditch `exec` completely and use `spawn`, I think.